src/Raven.Studio/wwwroot/App/views/database/indexes/editIndex.html

```diff
@@ -218,28 +215,24 @@ <h2 id="reduce-title" class="margin-bottom margin-bottom-xs on-base-background">
 <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ START AUTO INDEX ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
 <div data-bind="if: isAutoIndex, css: { 'content-margin': isAutoIndex }">
     <div class="flex-vertical edit-index auto-index">
-
         <div class="toolbar">
             <div class="row">
                 <div class="col-lg-8 col-xl-6 flex-horizontal">
                     <div class="input-group flex-grow" data-bind="validationElement: editedIndex().name">
                         <div class="input-group-addon">Index name</div>
                         <input class="form-control" data-bind="textInput: editedIndex().name" disabled>
                     </div>
-                    <div>
-                        <button class="btn btn-danger" data-bind="visible: isEditingExistingIndex, click: deleteIndex"><i class="icon-trash"></i> <span>Delete</span></button>
-                    </div>
                 </div>
                 <div class="col-lg-4 col-xl-6 flex-horizontal">
                     <div class="flex-separator"></div>
                     <div class="btn-group">
-                        <a data-bind="attr: { href: queryUrl }, visible: isEditingExistingIndex" class="btn" title="Query the index"><i class="icon-query"></i> <span>Query</span></a>
-                        <a data-bind="attr: { href: termsUrl }, visible: isEditingExistingIndex" class="btn" title="Navigate to index terms"><i class="icon-terms"></i> <span>Terms</span></a>
+                        <a data-bind="attr: { href: queryUrl }, visible: isEditingExistingIndex" class="btn" title="Query the index"><i class="icon-query"></i><span>Query</span></a>
+                        <a data-bind="attr: { href: termsUrl }, visible: isEditingExistingIndex" class="btn" title="Navigate to index terms"><i class="icon-terms"></i><span>Terms</span></a>
+                        <button class="btn btn-danger" data-bind="visible: isEditingExistingIndex, click: deleteIndex" title="Delete this Auto-Index"><i class="icon-trash"></i><span>Delete</span></button>
```

Comment:
title: Delete this index - I wouldn't care about index type here.

Reply:
done